Cornelica/scenario/rry_002.txt

```diff
@@ -3,156 +3,159 @@
 ;;好感度普通
 ;;===========================================================
 
-[リリィ]「その……アルって、Hは初めてだった？」
+[Lily]"Al... that was your first time?
 
-[リリィ]「わたしが初めてでよかった……？　魔力を吸われて、きつかったりしない？」
+[Lily]"I was good for the first time...? \nSiphon Charm, was hard?"
 
 ;;アルうなずく
 
-[リリィ]「そっか！　よかったぁ…………」
+[Lily]"Really! It was good..."
 
-[リリィ]「…………シたくなったら、いつでも言ってね？」
+[Lily]"When you want to do it, always say it ok?"
 
 ;;===========================================================
 
-[リリィ]「サキュバスってね、体のどこからでも、精液の『味』を感じることができるんだよ」
+[Lily]"Succubus can feel the semen's taste \nfrom anywhere in their body"
 
-[リリィ]「人間が飲んでも苦いだけらしいんだけど……サキュバスは相手によって違う味を感じるんだ」
+[Lily]"Even if for a human it seems bitter, \nSuccubus feels a different taste depending on the partner"
 
-[リリィ]「すっぱかったり、苦かったりするんだけど……アルのは、甘くておいしいよ♪」
+[Lily]"Can be sour or bitter...\nbut your is sweet and delicious♪♪"
 
 ;;===========================================================
+;;See rry_n003.txt!!!!!
+[Lily]"Why succubus's wings and tails \nare different from each other? \nThey change according to the strength of their magic"
 
-[リリィ]「サキュバスの羽や尻尾って、みんな少しづつ違うでしょ？　これって、それぞれの魔力の強さで変わったりするんだ」
+[Lily]"With lot of Magic Power, the shape change for make it a sucking cavity, and you can move those like hands. I can't do that yet..."
```

Comment:
yy, it miss a \n because I wasn't sure about the sucking cavity bit... with dictionary I wasn't able to find out another definition and I don't get it...